pkg/operator/controller/ingress/rsyslog_configmap.go

```diff
@@ -18,9 +18,13 @@ import (
 
 const (
 	// rsyslogConfiguration is the contents for rsyslog.conf.
+	// the parameter "$MaxMessageSize 10k" is needed to allow messages with
+	// 8192 characters, the default value allows messages with maximum 8096
+	// characters
```

Comment:
nit
```suggestion
	// The parameter "$MaxMessageSize 10k" is needed to allow messages with
	// 8192 characters, because the default value only allows messages with maximum 
	// 8096 characters.
```

Reply:
done